checker/unneeded_import_alias_checker.go

```diff
@@ -0,0 +1,103 @@
+package checker
```

Comment:
Correct me if I'm wrong but the checker will fail in the following case:

```go
import (
    "gopkg.in/yaml.v2"
    myyaml "github.com/myrepos/yaml"
)
```

Reply:
Updated to use package names instead of import names with latest commit.